Mathlib/SetTheory/Cardinal/Regular.lean

```diff
@@ -0,0 +1,80 @@
+/-
+Copyright (c) 2018 Nir Paz. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Nir Paz
+-/
+import Mathlib.SetTheory.Cardinal.Cofinality
+
+/-!
+# Regular cardinals
+
+We define singular cardinals and prove theorems about singular and regular cardinals.
+
+## Main definitions and results
+
+* `Cardinal.IsSingular c` means that `c` is an infinite cardinal which is not regular. That is,
+  its cofinality is smaller than itself.
+-/
+
+
+open Ordinal Cardinal
+
+section Singular
```

Comment:
This section should be inside the cardinal namespace. That way you won't have to move it when you introduce the Regular section.